src/components/PlaylistPage/LessonItem.js

```diff
@@ -20,22 +23,31 @@ export const LessonItem = React.createClass({
         </Button>
       </LinkContainer>
       : null;
+    const tooltipContent = getLessonIntro(lesson.path.slice(1));
+    const tooltip =
+      <Tooltip id="lessonIntro">
+        <div dangerouslySetInnerHTML={{__html: tooltipContent}} />
+      </Tooltip>;
     return (
       lesson.external ?
-        <ListGroupItem href={lesson.external} target="_blank" className={styles.row}>
-          {levelIcon}
-          <div className={styles.title}>{lesson.title}</div>
-          {instructionBtn}
-          &nbsp;<Glyphicon glyph="new-window"/>
-        </ListGroupItem>
-        :
-        <LinkContainer to={lesson.path}>
-          <ListGroupItem className={styles.row}>
+        <OverlayTrigger animation={true} delayShow={400} placement="bottom" overlay={tooltip}>
```

Comment:
Det ser ut som om det er samme OverlayTrigger som legges på for internal og external. Kan man ikke bare ha den én gang (før `lesson.external ?`)?